packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -1007,4 +1007,14 @@ whisperkit-cli transcribe --audio-path /path/to/audio.mp3
 # Or use your preferred model variant
 whisperkit-cli transcribe --model "large-v3" --model-prefix "distil" --audio-path /path/to/audio.mp3 --verbose`,
 ];
+
+export const threedtopia_xl = (): string[] => [
```

Comment:
```suggestion
export const threedtopia_xl = (model: ModelData): string[] => [
```

you need the model data as input to correctly generate the snippet :)

Reply:
Thanks! Updated